test/python/circuit/library/test_evolution_gate.py

```diff
@@ -113,6 +113,25 @@ def test_suzuki_trotter_manual(self):
 
         self.assertEqual(evo_gate.definition.decompose(), expected)
 
+    def test_qdrift_manual(self):
+        """Test the evolution circuit of Suzuki Trotter against a manually constructed circuit."""
+        op = X + Y
+        time = 0.5
+        reps = 1
+        qdrift = QDrift(reps=reps)
+        evo_gate = PauliEvolutionGate(op, time, synthesis=qdrift)
+        evo_gate.definition.decompose()
+
+        # manually construct expected evolution
+        expected = QuantumCircuit(1)
+        for pauli in qdrift.sampled_ops:
+            if pauli[0].to_label() == "X":
+                expected.rx(2 * pauli[1], 0)
+            elif pauli[0].to_label() == "Y":
+                expected.ry(2 * pauli[1], 0)
+
+        self.assertEqual(evo_gate.definition.decompose(), expected)
+
```

Comment:
Maybe an additional unit test with `reps = 2`? Also, maybe an additional one with a degenerate operator, like `op = X`?

Reply:
added a new test with `reps=2` and `op=X`

Reply:
Thanks!